helm/templates/storage/README.md

```diff
@@ -0,0 +1,12 @@
+# SBOMbastic Storage
+
+The `storage` Helm chart installs the SBOMbastic storage deployment, which should be installed alongside the SBOMbastic controller and worker components.
+
+The storage component uses SQLite as its database backend. To ensure data persistence, it requires a PersistentVolumeClaim (PVC) backed by a PersistentVolume (PV). Users need to either:
+
+1. Have a StorageClass configured that can dynamically provision PVs, or
+2. Manually create a PV that matches the PVC requirements
```

Comment:
Usually charts let the user create the PVC on their own, with their own criteria, and the provide that to the helm chart as a value.

I'll provide more details later